dvc/config.py

```diff
@@ -392,7 +387,7 @@ def load(self):
         try:
             d = self.COMPILED_SCHEMA(d)
         except Invalid as exc:
-            raise ConfigError(str(exc), cause=exc)
+            raise ConfigError(str(exc)) from exc
```

Comment:
Python 3 chains them implicitly, so this is the same as not saying `from ...` .

Reply:
Discussed this privately, to me explicit makes more sense, since it will also provide a traceback with "was caused by" message instead of "during handling of exception another exception occured".